### DISCUSSION TITLE
Panoramic photos cannot recognize faces

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#13336] [Q&A]
### The bug

Open a 360 degree panoramic photo containing a face, and the face recognition prompt says' Error loading image ', unable to recognize the face.
![image](https://github.com/user-attachments/assets/add711ca-f9f9-43ca-8f17-b9ec18fc050a)
![image](https://github.com/user-attachments/assets/547f6381-462a-4827-9bd0-6c702063f3e7)

### The OS that Immich Server is running on

Windows 11 Docker Desktop V4.34.3

### Version of Immich Server

V1.117.0

### Version of Immich Mobile App

V1.117.0

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not related
```

### Your .env content

```Shell
not related
```

### Reproduction steps

1.
2.
3.
...

**Comment by DeltaTango69** [comment]
This is working for me.

**Reply by pyccl** [reply]
I have over 7000 photos in total, including about 20 panoramic photos. Among them, there are only two or three photos with faces, but neither of them is normal as they do not have thumbnail images of faces.
